.npmignore

```diff
@@ -0,0 +1,9 @@
+node_modules*
```

Comment:
❤️

Reply:
Thank you for the .npmignore introduction ;)